scripts/setup/mac/install_prereqs

```diff
@@ -50,20 +50,22 @@ if [[ ! -f /usr/include/expat.h || ! -f /usr/include/zlib.h ]]; then
   exit 3
 fi
 
-brew tap dreal-deps/coinor
+brew tap dreal-deps/ibex
```

Comment:
FYI, this line is not necessary (as we do not need `dreal-deps/coinor`). They are included by `brew tap dreal/dreal` and I think it's clearer in that way.

Reply:
The behavior of fully-qualified names in `depends_on` is actually inconsistent. We stopped using them in `robotlocomotion/director` as they were causing issues.

Reply:
I see. Could you drop a link to an upstream resource about it if you have any?